### ISSUE TITLE
[$10][MSFT-252] "Continue" button foreground and background colors missed WCAG 2 AA contrast ratio

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2904]
**Steps to Reproduce**
1. Go to: https://accounts.topcoder.com/sso-login/?app=member
2. Right click > Inspect Elements 
3. Check the "Continue" button

**Expected Result**
- Contrast ratio should be 4.5:1
- If you want passes WCAG 2 AA, you can change Background Color to #0072F0 Or you can change text color to #011565. New contrast ratio: 4.5:1

**Actual Result**
Foreground Color: #FFFFFF
Background Color: #1A85FF
Contrast Ratio: 3.59:1
=> Therefor, "Continue" button foreground and background colors missed WCAG 2 AA contrast ratio

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F31

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096231 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096231 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096231 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/appirio-tech/accounts-app/pull/222

**Comment by Dara-K** [comment]
Please use #006AD7 as the new blue color for the button.

**Comment by PrakashDurlabhji** [comment]
@Dara-K @codeMinter PR updated according to above

**Comment by tosha5252** [comment]
Button color changed to #006AD7

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096727 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096727 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by lakshmiathreya** [comment]
Verified Continue button set to #006AD7. Tested on Win10/Chrome & MacOS/Chrome&Firefox.

**Comment by cwdcwd** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096727<br/><br/>```This is an automated message for lazybaer via Topcoder X```
